config/zones/US-CAL-CISO.yaml

```diff
@@ -46,21 +46,33 @@ emissionFactors:
         source: Electricity Maps, 2021 average
         value: 251.31712032739068
     biomass:
-      datetime: '2020-01-01'
-      source: eGrid 2020
-      value: 0.0
+      - datetime: '2020-01-01'
+        source: eGrid 2020
+        value: 0.0
+      - datetime: '2022-01-01'
+        source: eGRID 2021
+        value: 0.0
     coal:
-      datetime: '2020-01-01'
-      source: eGrid 2020
-      value: 514.4743088
+      - datetime: '2020-01-01'
+        source: eGrid 2020
+        value: 514.4743088
+      - datetime: '2022-01-01'
+        source: eGRID 2021
+        value: 525.49
     gas:
-      datetime: '2021-01-01'
-      source: eGrid 2020
-      value: 403.8432591
+      - datetime: '2021-01-01'
+        source: eGrid 2020
+        value: 403.8432591
+      - datetime: '2022-01-01'
+        source: eGRID 2021
+        value: 388.81
```

Comment:
These use the same source but have different datetime, is this intentional?

Reply:
I fixed it last night, I don't know what happened